unifi.go

```diff
@@ -142,6 +142,13 @@ func (u *Unifi) Login() error {
 	return nil
 }
 
+// Logout closes the current session
+func (u *Unifi) Logout() error {
+	var response struct {
+	}
+	return u.GetData(APILogoutPath, &response)
+}
```

Comment:
```suggestion
// Logout closes the current session
func (u *Unifi) Logout() error {
	var response interface{}
	return u.GetData(APILogoutPath, &response)
}
```

I think this may be more consistent, but not a requirement.